src/UserSpaceInstrumentation/UserSpaceInstrumentationTestLib.h

```diff
@@ -11,13 +11,16 @@
 // injects a binary produced by this code into its child.
 
 // Call first to initialize the library.
-void InitTestLib();
+extern "C" void InitTestLib();
 
 // Log a string into a temporary file.
-void UseTestLib(std::string_view s);
+extern "C" void UseTestLib(std::string_view s);
```

Comment:
you might also want to make it const char* if you want a pure C interface

Reply:
Yep, might make things easier. done.